src/api/db/schemas/Project.js

```diff
@@ -73,7 +81,8 @@ const ProjectSchema = new Schema({
   views: { type: [ViewSchema], required: true },
   cameraConfigs: { type: [CameraConfigSchema] },
   availableMLModels: { type: [{ type: String, ref: 'MLModel' }] },
-  automationRules: { type: [AutomationRuleSchema] }
+  automationRules: { type: [AutomationRuleSchema] },
+  labels: { type: [ProjectLabelSchema], default: [], required: true }
```

Comment:
@ingalls, can you add 2 default labels here? One for 'empty' (`color: '#8D8D8D'`) and one for 'unknown' (`color: '#E93D82'`)?